<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>dspInventoryAvailabilityBySalesOrder</class>
 <widget class="QWidget" name="dspInventoryAvailabilityBySalesOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inventory Availability by Sales Order</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="_salesOrderLit">
         <property name="text">
          <string>Sales Order #:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="OrderCluster" name="_so">
         <property name="label">
          <string/>
         </property>
         <property name="nameVisible">
          <bool>false</bool>
         </property>
         <property name="descriptionVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="_orderDateLit">
         <property name="text">
          <string>Order Date:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>_orderDate</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="DLineEdit" name="_orderDate">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout">
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLabel" name="_custNameLit">
           <property name="text">
            <string>Cust. Name:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="_custPhoneLit">
           <property name="text">
            <string>Cust. Phone:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="_poNumberLit">
           <property name="text">
            <string>P/O Number:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLabel" name="_custName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="_custPhone">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="_poNumber">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout">
     <item>
      <widget class="XCheckBox" name="_onlyShowShortages">
       <property name="text">
        <string>Only Show Shortages</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XCheckBox" name="_useReservationNetting">
       <property name="text">
        <string>Use Reservation Netting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XCheckBox" name="_showWoSupply">
       <property name="text">
        <string>Show W/O Supply</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="5"/>
 <customwidgets>
  <customwidget>
   <class>DLineEdit</class>
   <extends>QWidget</extends>
   <header>datecluster.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OrderCluster</class>
   <extends>QWidget</extends>
   <header>ordercluster.h</header>
  </customwidget>
  <customwidget>
   <class>XCheckBox</class>
   <extends>QCheckBox</extends>
   <header>xcheckbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_onlyShowShortages</tabstop>
  <tabstop>_useReservationNetting</tabstop>
  <tabstop>_showWoSupply</tabstop>
  <tabstop>_orderDate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
